.github/workflows/conformance-eks-v1.10.yaml

```diff
@@ -269,7 +269,11 @@ jobs:
 
       - name: Run connectivity test
         run: |
-          cilium connectivity test --force-deploy --flow-validation=disabled
+          cilium connectivity test --force-deploy --flow-validation=disabled \
+            --test '!client-egress-l7,!echo-ingress-l7'
+      # workaround for L7 tests issues on EKS with IPsec enabled
+      # TODO: remove both test exceptions once:
+      # - https://github.com/cilium/cilium/issues/17139 is fixed
```

Comment:
Did we check the regression also concerns v1.10?

Reply:
Not yet -- I will do so as part of triaging. I've decided to include it anyway so as to avoid a second PR if I confirm it's present there as well.